tests/vpp.cpp

```diff
@@ -21,18 +21,22 @@
 #include "vppoutputencode.h"
 #include "encodeinput.h"
 #include "common/log.h"
+#include "tests/vppinputasync.h"
 #include <Yami.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <getopt.h>
 #include <string.h>
+#include <deque>
 
 using namespace YamiMediaCodec;
 
 void usage();
 
-SharedPtr<VppInput> createInput(const char* filename, const SharedPtr<VADisplay>& display)
+SharedPtr<VppInput> createInput(const char* filename, const SharedPtr<VADisplay>& display, uint32_t startupSize)
```

Comment:
The startupSize is frame number, char is not enough for it, since we may have more than 1000 frames.